src/Compilers/Core/Portable/ReferenceManager/CommonReferenceManager.Binding.cs

```diff
@@ -17,14 +18,25 @@ internal partial class CommonReferenceManager<TCompilation, TAssemblySymbol>
         ///
         /// The first element (index==0) of the assemblies array represents the assembly being built.
         /// One can think about the rest of the items in assemblies array as assembly references given to the compiler to
-        /// 
         /// build executable for the assembly being built. 
-        /// 
         /// </summary>
         /// 
         /// <param name="assemblies">
-        /// The set of <see cref="AssemblyData"/> objects describing assemblies, for which this method should
+        /// Input: An array of <see cref="AssemblyData"/> objects describing assemblies, for which this method should
         /// resolve references and find suitable AssemblySymbols.
+        /// Output: Updated array with resolved implicitly referenced assemblies appended.
+        /// </param>
+        /// <param name="resolverOpt">
+        /// Reference resolver used to look up missing assemblies.
+        /// </param>
+        /// <param name="importOptions">
+        /// Import options applied to implicitly resolved references.
+        /// </param>
+        /// <param name="implicitlyResolvedReferences">
+        /// Implicitly resolved references, corresponding to <see cref="AssemblyData"/> appended to the <paramref name="assemblies"/> array.
```

Comment:
Where this implicitly resolved references? Are they the missing references resolved by resolverOpt? If so, perhaps it is worth clarifying whether this builder is supposed to be empty on entry and be populated by the method, or it already can contain something (BTW, where that something came from?), etc.